grafana_client/api.py

```diff
@@ -121,7 +121,7 @@ def connect(self):
     def version(self):
         if not self._grafana_info:
             self._grafana_info = self.health.check()
-        version = self._grafana_info["version"]
+        version = self._grafana_info.get("version", "unknown")
```

Comment:
Would returning `None` also be an option? It would be a more agnostic token for downstream users to act upon, for example when parsing the version string using [verlib2](https://pypi.org/project/verlib2/), or friends.